sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,90 @@
+import atexit
+from multiprocessing import Process
+import numpy as np
+import pickle
+import platform
+from queue import Empty, Queue
+import tensorflow as tf
+from threading import Thread
+
+from rllab.sampler.utils import rollout
+
+__all__ = ['init_worker', 'init_plot', 'update_plot']
+
+process = None
```

Comment:
Encapsulate the variables and the methods in this file within a class to avoid having global variables.